presto-main/src/main/java/com/facebook/presto/operator/DriverContext.java

```diff
@@ -175,6 +177,11 @@ public void failed(Throwable cause)
         finished.set(true);
     }
 
+    public void close()
```

Comment:
consider implementing `AutoCloseable` to let users know the `close()` must be called